WMIAdventure/backend/WMIAdventure_backend/battle/businesslogic/card_states/CardState.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any
+
+
+class CardState:
+    """
+    This is abstract class of card's state. Concrete card state classes should derive from this class.
+    Card can have states like being blocked.
+    """
+    turns_active: int
+    active: bool
+
+    def __init__(self, turns_active: int):
```

Comment:
Może ustawić turns_active domyślnie na 1? Większość efektów będzie trwała turę, więc warto oszczędzić przyszłym koderom pisania zawsze tego parametru.

Ogólnie to tutaj pojawia się też problem skąd wziąć ilość tur jeżeli chodzi o serializację itp, więc taki domyślny parametr zamiata problem pod dywan

Reply:
Jeśli chodzi o serializację to nie widzę problemu, serializer po prostu weźmie pole turns_active